deprecated/README.md

```diff
@@ -1,8 +1,13 @@
 # Deprecated Deployments
 
-This folder contains deprecated versions of 1Password SCIM bridge deployments. 
+This folder contains 1Password SCIM Bridge deployment methods that have been deprecated. At the time of deprecation, these deployments are still fully functional, but will no longer be updated.
 
-At the time of deprecation, these deployments are still fully functional, but may no longer be updated and will eventually be removed.
+> 💡 **Note** that it is solely the _deployment method_ that is deprecated. Deprecating a deployment method is independent of the 1Password SCIM Bridge itself, or a specific version of the 1Password SCIM Bridge. For information about the latest version of 1Password SCIM Bridge, please see the [changelog](https://app-updates.agilebits.com/product_history/SCIM).
 
 ## Deployments
- - [aws-terraform](./aws-terraform) (2020-12-21)
+
+| Deployment                                                | Deprecation Date | Deletion Date | Suggested Alternative                                                                                                                |
+| --------------------------------------------------------- | ---------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
+| [aws-ec2-terraform](./aws-terraform)                      | 2020-12-21       | 2023-09-14    | [AWS ECS Fargate with Terraform](../aws-ecsfargate-terraform/) or [AWS ECS Fargate with CloudFormation](./beta/aws-ecsfargate-cfn/)  |
+| [DigitalOcean App Platform](./digitalocean-app-platform/) | 2022-12-21       | 2023-09-14    | [Digital Ocean App Platform with `op` CLI](../beta/do-app-platform-op-cli/) or [Azure Container Apps](../beta/azure-container-apps/) |
+
```

Comment:
I'd add one other section to this document: guidance on how to deprecate a deployment for other Solution Architects to follow. 

EG:
1. Determine if we want to deprecate a deployment method 'baz' based on _ and _
1. Open a branch `deprecate/baz`
1. Move `baz` into `deprecated/baz`
1. Calculate a convenient deletion date approximately three months from now. A good day would be a workday without immediate holidays around it.
1. Add today's date to the `deployments` table
1. Add the deletion date to the `deployments` table
1. Put your MR up for review and approval

I'd also include exact instructions on how to do a deletion.

Reply:
Resolved in [92899a5](https://github.com/1Password/scim-examples/pull/249/commits/92899a5be260e7e08002ddd4dc1b1152648fefcc)